### PR TITLE
API: Transaction | Create new case 

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Report:
Screen Shot 2023-05-25 at 17 12 17

File: src/main/java/microservices/txn/steps/TransactionSteps.java [hunk 1]
```diff
@@ -59,8 +55,37 @@ public TransactionSteps then_verifyFilterTransactionResponse(FilterTransactionRe
...
 
+    @Step("Get current transaction")
+    public TransactionSteps when_getCurrentTransaction() {
+        sendGet(Endpoints.TransactionApi.TRANSACTION_CURRENT);
+        return this;
+    }
+
+    @Step("Get current transaction")
+    public TransactionSteps then_verifyCurrentTransactionResponse(List<Transaction> currentTransactionsList) {
+        Assert.assertEquals(currentTransactionsList.size(),4);
```
Comment (Sotatek-GiangNguyen7) [inline src/main/java/microservices/txn/steps/TransactionSteps.java:75]: size of this is always 4?

File: src/main/java/microservices/txn/steps/TransactionSteps.java [hunk 2]
```diff
@@ -59,8 +55,37 @@ public TransactionSteps then_verifyFilterTransactionResponse(FilterTransactionRe
...
+        return this;
+    }
+
+    @Step("Verify transaction graph response")
+    public TransactionSteps then_verifyTypeTransactionResponse(List<TransactionGraphResponse> transactionGraphResponseList, int day) {
+        String endDate = DateUtil.getCurrentUTCDate(DATE_FORMAT[1]);
+        System.out.println(endDate);
```
Comment (Sotatek-GiangNguyen7) [inline src/main/java/microservices/txn/steps/TransactionSteps.java:82]: delete

File: src/main/java/microservices/txn/steps/TransactionSteps.java [hunk 3]
```diff
@@ -59,8 +55,37 @@ public TransactionSteps then_verifyFilterTransactionResponse(FilterTransactionRe
...
+        String startDate = DateUtil.getCurrentUTCSubDays(day, DATE_FORMAT[1]);
+        System.out.println(startDate);
```
Comment (Sotatek-GiangNguyen7) [inline src/main/java/microservices/txn/steps/TransactionSteps.java:84]: delete

**Review by Sotatek-GiangNguyen7 — APPROVED** [review]

**Pull request merged**